share/picongpu/examples/FoilLCT/etc/picongpu/4_isaac.cfg

```diff
@@ -46,7 +46,7 @@ TBG_periodic="--periodic 0 0 0"
 ## Section: Optional Variables ##
 #################################
 
-TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hypnos5 --isaac.quality 99"
+TBG_isaac="--isaac.period 1 --isaac.name foil --isaac.url hemera4 --isaac.quality 99"
```

Comment:
This requires that the isaac server will work on hemera - so far, we could only build it for hypnos. @psychocoderHPC Have you compiled the isaac server on hemera?

Reply:
This is only an example and hypnos is on the way to get disabled.

Reply:
The bug that it is not building on hemera must be fixed in ISACC.

Reply:
@PrometheusPi We have few open issue in ISAAC if it is a compile issue than please open a new issue there if it is the issue with [dumping png](https://github.com/ComputationalRadiationPhysics/isaac/issues/95) there is no new issue needed.

Reply:
I am not sure whether it does not build on hemera. On taurus it build fine but did not work - thus we will have to check this. But this is only a minor adjustment, thus I am fine with changing this to hemera.